config/env_config.py

```diff
@@ -16,6 +16,12 @@ def __init__(self):
         self.discord_webhook_url = get_secret(
             "boost-bot/prod-discord-url", "DISCORD_WEBHOOK_URL", test=self.test
         )
+
+        self.explorer_api_keys = {
+            "eth": get_secret("", "ETHERSCAN_API_KEY", test=self.test),
```

Comment:
keepers/etherscan can be used for first param, and then can do polygonscan, arbiscan for the other 2

Reply:
Also the key is ETHERSCAN_TOKEN